tests/unit/source_code/test_files.py

```diff
@@ -5,37 +5,37 @@
 from databricks.sdk.service.workspace import Language
 
 from databricks.labs.ucx.hive_metastore.migration_status import MigrationIndex
-from databricks.labs.ucx.source_code.files import LocalFileMigrator, LocalFileResolver, FileLoader
+from databricks.labs.ucx.source_code.files import LocalFilesMigrator, LocalFileResolver, FileLoader
```

Comment:
doesn't seem to be relevant to this PR, revert the rename.

Reply:
done

Reply:
done